Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["bin/faucet"]
+members = ["bin/faucet", "crates/client"]
```

Comment:
Not sure about using "client" here still. I think we could also just call it "faucet". So, this line could be:
```toml
members = ["bin/faucet", "crates/faucet"]
```
Or maybe
```toml
members = ["bin/faucet", "crates/faucet-lib"]
```